transport-classes-epoll/src/main/java/io/netty/channel/epoll/EpollEventLoop.java

```diff
@@ -102,6 +103,12 @@ public int get() throws Exception {
             allowGrowing = false;
             events = new EpollEventArray(maxEvents);
         }
+        openFileDescriptors();
+    }
+
+    // This method is intended for use by OpenJDK CRaC integration.
```

Comment:
Consider turning these comments into javadoc comments, and make them more generic, i.e. something along the lines of "intended for use by a process checkpoint/restore integration, such as OpenJDK CRaC".

Reply:
OK, out of curiosity, does @UnstableAPI still get to Javadocs?